benches/bin/mvcctxn.rs

```diff
@@ -0,0 +1,472 @@
+// Copyright 2017 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+extern crate rand;
+
+use std::time::Instant;
+use std::sync::mpsc::channel;
+use std::sync::Arc;
+
+use tikv::storage::{Key, Modify, Mutation, Options, Snapshot, SnapshotStore, Statistics,
+                    CF_DEFAULT, CF_LOCK, CF_RAFT, CF_WRITE};
+use tikv::storage::mvcc::MvccTxn;
+use tikv::raftstore::store::engine::SyncSnapshot;
+use tikv::config::DbConfig;
+use tikv::util::threadpool::{DefaultContext, ThreadPoolBuilder};
+use tikv::util::config::ReadableSize;
+use tikv::util::rocksdb::{get_cf_handle, new_engine_opt, CFOptions};
+use kvproto::kvrpcpb::IsolationLevel;
+use rocksdb::{Writable, WriteBatch, DB};
+use rocksdb::rocksdb_options::DBOptions;
+
+//use super::print_result;
+//use test::BenchSamples;
+
+use rand::Rng;
+use tempdir::TempDir;
+
+use utils::*;
+
+
+
+#[inline]
+fn do_write(db: &DB, modifies: Vec<Modify>) {
+    let wb = WriteBatch::new();
+    for rev in modifies {
+        match rev {
+            Modify::Delete(cf, k) => if cf == CF_DEFAULT {
```

Comment:
You can get `CFHandle` of `CF_DEFAULT` too, so it is not necessary to distinguish them.